index.html

```diff
@@ -36,7 +36,7 @@
         <div class="pl-4 flex items-center">
           <a class="toggleColour text-black no-underline hover:no-underline font-bold text-2xl lg:text-2xl" href="#">
             <!--Icon from: http://www.potlabicons.com/ -->
-            Web3 Community
+            <img src="./logo.png" alt="Web3 Community" class="h-16 items-start">
```

Comment:
Add space at line 39 "Web 3 Community"

Reply:
Yess

Reply:
Fixed it with https://github.com/mkubdev/web3community.github.io/commit/5b64ecfc3d2f406b4d6ba88e6813c681ba620334